src/expr/node_manager.h

```diff
@@ -444,6 +444,9 @@ class NodeManager {
   /** Get a Kind from an operator expression */
   static inline Kind operatorToKind(TNode n);
 
+  /** Get corresponding application kindfor function */
```

Comment:
kindfor --> kind for.
Also can you give some documentation for why this function is necessary?